### PR TITLE
Fix for RM worker and update 2 allocate wfs

### PR DESCRIPTION
**Pull request by Branislav-Karkus**
- Change version to 1.1.5
- Fix for update_alt_id task in resource manager worker
- Change input params value in 2 allocate WFs

### CODE REVIEW

File: frinx_conductor_workers/frinx_conductor_workflows/resource_manager_workflows/Allocate_IPv6_Prefix_Resource.json [hunk 1]
```diff
@@ -135,7 +135,7 @@
     }
   ],
   "inputParameters": [
-    "{\"pool_name\":{\"value\":\"\",\"description\":\"Pool name\",\"type\":\"string\"}, \"description\":{\"value\":\"\",\"description\":\"[OPTIONAL] Description for resource\",\"type\":\"string\"}, \"prefix\":{\"value\":\"24\",\"description\":\"Subnet prefix\",\"type\":\"string\"}, \"desired_value\":{\"value\":\"\",\"description\":\"[OPTIONAL] Desired value ip address (for example 2001:db8:1::)\",\"type\":\"string\"}}"
```
Comment (Vasyl-9) [inline frinx_conductor_workers/frinx_conductor_workflows/resource_manager_workflows/Allocate_IPv6_Prefix_Resource.json:138]: Why you delete this example value?

File: frinx_conductor_workers/frinx_conductor_workflows/resource_manager_workflows/Allocate_IPv4_Prefix_Resource.json [hunk 2]
```diff
@@ -135,7 +135,7 @@
       }
     ],
     "inputParameters": [
-        "{\"pool_name\":{\"value\":\"\",\"description\":\"Pool name\",\"type\":\"string\"}, \"description\":{\"value\":\"\",\"description\":\"[OPTIONAL] Description for resource\",\"type\":\"string\"}, \"prefix\":{\"value\":\"24\",\"description\":\"Subnet prefix\",\"type\":\"string\"}, \"desired_value\":{\"value\":\"\",\"description\":\"[OPTIONAL] Desired value ip address (for example 192.168.0.0)\",\"type\":\"string\"}}"
+        "{\"pool_name\":{\"value\":\"\",\"description\":\"Pool name\",\"type\":\"string\"}, \"description\":{\"value\":\"\",\"description\":\"[OPTIONAL] Description for resource\",\"type\":\"string\"}, \"prefix\":{\"value\":\"\",\"description\":\"Subnet prefix\",\"type\":\"string\"}, \"desired_value\":{\"value\":\"\",\"description\":\"[OPTIONAL] Desired value ip address (for example 192.168.0.0)\",\"type\":\"string\"}}"
```
Comment (Vasyl-9) [inline frinx_conductor_workers/frinx_conductor_workflows/resource_manager_workflows/Allocate_IPv4_Prefix_Resource.json:138]: Why you delete this example value?

**Review by Vasyl-9 — CHANGES_REQUESTED** [review]

File: frinx_conductor_workers/frinx_conductor_workflows/resource_manager_workflows/Allocate_IPv6_Prefix_Resource.json (hunk 1 above)
Comment (Branislav-Karkus) [reply frinx_conductor_workers/frinx_conductor_workflows/resource_manager_workflows/Allocate_IPv6_Prefix_Resource.json]: In new commit is this change reverted

File: frinx_conductor_workers/frinx_conductor_workflows/resource_manager_workflows/Allocate_IPv4_Prefix_Resource.json (hunk 2 above)
Comment (Branislav-Karkus) [reply frinx_conductor_workers/frinx_conductor_workflows/resource_manager_workflows/Allocate_IPv4_Prefix_Resource.json]: In new commit is this change reverted

**Review by Vasyl-9 — CHANGES_REQUESTED** [review]

File: frinx_conductor_workers/frinx_conductor_workers/resource_manager_worker.py [hunk 3]
```diff
@@ -1381,16 +1389,16 @@ def query_recently_active_resources(task, logs):
...
     """
-    fromDatetime = task["inputData"]["fromDatetime"]
-    toDatetime = task["inputData"]["toDatetime"] if "toDatetime" in task["inputData"] else None
+    from_datetime = task["inputData"]["fromDatetime"]
+    to_datetime = task["inputData"]["toDatetime"] if "toDatetime" in task["inputData"] else None
     first = task["inputData"]["first"] if "first" in task["inputData"] else None
     last = task["inputData"]["last"] if "last" in task["inputData"] else None
     before = task["inputData"]["before"] if "before" in task["inputData"] else None
     after = task["inputData"]["after"] if "after" in task["inputData"] else None
     body = query_recently_active_resources_template.render()
```
[the remark this answers is not in the document]
Comment (Branislav-Karkus) [reply frinx_conductor_workers/frinx_conductor_workers/resource_manager_worker.py:1407]: Done

**Review by Vasyl-9 — APPROVED** [review]

**Pull request merged**